### PR TITLE
add leaflet-omnivore v0.3.2

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #6746
@dannyxx001  please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mapbox/leaflet-omnivore
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/leaflet-omnivore
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 254
   - Count of watchers: 102
   - Count of forks: 63
 * NPM download stats (optional):
   - Downloads in the last day: 29
   - Downloads in the last week: 98
   - Downloads in the last month: 452

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

npm auto-update config added, cc #6746, cc @tmcw

### CODE REVIEW

File: ajax/libs/leaflet-omnivore/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "gpx",
+    "wkt",
+    "osm",
+    "polyline",
+    "topojson",
+    "format",
+    "converter"
+  ],
+  "author": "Tom MacWright",
+  "license": "BSD",
```
Comment (PeterDaveHello) [inline ajax/libs/leaflet-omnivore/package.json]: What kind of BSD? Please confirm that.

**Comment by dannyxx001** [comment]
And the autoupdate config had better be put in the last line.

**Comment by kennynaoh** [comment]
@dannyxx001 
Please help me review this PR.
Thanks.

**Comment by dannyxx001** [comment]
@PeterDaveHello I think this PR is OK. Please review again, thanks.

**Comment by PeterDaveHello** [comment]
The license is not the same as official package.json, have you confirmed that? I didn't see you mention any related issues.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
Sorry, I didn't mention it.
I have confirmed with author by this [issue](https://github.com/mapbox/leaflet-omnivore/issues/79)
Help me review it again.
Thanks.

**Comment by PeterDaveHello** [comment]
@kennynaoh so you should also send a PR to contribute to upstream:
https://github.com/mapbox/leaflet-omnivore/blob/master/package.json#L40

**Comment by kennynaoh** [comment]
@PeterDaveHello 
OK, thanks.

**Pull request merged**